tests/testthat/_snaps/upkeep.md

```diff
@@ -68,4 +68,45 @@
       * [ ] `use_standalone("r-lib/rlang", "types-check")` instead of home grown argument checkers
       * [ ] Add alt-text to pictures, plots, etc; see https://posit.co/blog/knitr-fig-alt/ for examples
       
+      <sup>Created on DATE with `usethis::use_tidy_upkeep_issue()`, using [usethis vVERSION](https://usethis.r-lib.org)</sup>
+
+# upkeep bullets don't change accidentally
+
+    Code
+      writeLines(upkeep_checklist())
+    Output
+      * [ ] `usethis::use_readme_rmd()`
+      * [ ] `usethis::use_github_links()`
+      * [ ] `usethis::use_pkgdown_github_pages()`
+      * [ ] `usethis::use_package_doc()`
+      Consider letting usethis manage your `@importFrom` directives here. `usethis::use_import_from()` is handy for this.
```

Comment:
Is this indented correctly?

Reply:
It does render correctly in the issue (starts a newline as part of the same bullet): https://github.com/ateucher/rmapshaper/issues/155